src/main/java/com/atlassian/jira/cloud/jenkins/listeners/AutoBuildsListener.java

```diff
@@ -151,8 +171,7 @@ private boolean matchesRegex(final String autoBuildsRegex, final String displayN
             return Pattern.compile(autoBuildsRegex).matcher(displayName).matches();
         } catch (final PatternSyntaxException exception) {
             final String message = "PatternSyntaxException: " + exception.getMessage();
-            pipelineLogger.warn(message);
-            systemLogger.warn(message, exception);
```

Comment:
Removed the `systemLogger` completely here, as it logs to the Jenkins server logs, which are not really helpful to diagnose our customers' issues.